src/parser/mod.rs

```diff
@@ -8470,11 +8470,24 @@ impl<'a> Parser<'a> {
         })
     }
 
+    fn parse_pragma_value(&mut self) -> Result<Value, ParserError> {
+        match self.parse_value()? {
+            v @ Value::SingleQuotedString(_) => Ok(v),
+            v @ Value::DoubleQuotedString(_) => Ok(v),
+            v @ Value::Number(_, _) => Ok(v),
+            v @ Value::Placeholder(_) => Ok(v),
```

Comment:
Is there test coverage for placeholders? If not, can you pleaes add a test that uses a placeholder so we don't accidentally introduce a regression in the future?

Reply:
Totally agree, I don't think it'll introduce a regression as this case was not covered before as well, I've added a test for `table_info = ?`, could not make `table_info($)` work as it does not parse correctly, if you think this case should also be covered I would like a suggestion on how to write it, as `table_info('$')` does not look correct because placeholder usually are not surrounded by quotes.

Let me know it if suffices.

Reply:
> Totally agree, I don't think it'll introduce a regression as this case was not covered before as well

yes, I agree -- sorry for my poor wording. What I was trying to say was that without the additional test for `?` someone else in the future might introduce a regression (break the parsing of `?`) without any tests failing.